modules/claims_api/app/controllers/claims_api/v2/veterans/evidence_waiver_controller.rb

```diff
@@ -7,7 +7,28 @@ class EvidenceWaiverController < ClaimsApi::V2::ApplicationController
         before_action :verify_access!
 
         def submit
-          render json: { status: 'OK' }
+          bgs_claim = find_bgs_claim!(claim_id: params[:id])
+
+          claim_empty = bgs_claim&.dig(:benefit_claim_details_dto).nil? && bgs_claim&.dig(:bnft_claim_dto).nil?
```

Comment:
how did you find the `:benefit_claim_details_dto`? If I pass a valid claim ID I get
```
(byebug) bgs_claim.keys
[:bnft_claim_dto, :"@xmlns:ns0"]
```

Invalid ID provides the same..
```
(byebug) bgs_claim
{:bnft_claim_dto=>nil, :"@xmlns:ns0"=>"http://benefitclaim.services/"}
(byebug) bgs_claim.keys
[:bnft_claim_dto, :"@xmlns:ns0"]
```

an additional concern here is handling the error from BGS.... 
test this with an invalid claim ID with letters, like `123abc` and you get a big ol' Java Exception.. which we're sending back currently.

```
"title": "Internal server error",
            "detail": "Internal server error",
            "code": "500",
            "status": "500",
            "meta": {
                "exception": "(ns0:Server) \nException Description: The object [123abc], of class [class java.lang.String]..........
}
```

Reply:
In regards to `benefit_claim_details_dto`, it was due to poor test setup, and is now resolved.